src/Workspaces/Core/Portable/SymbolSearch/ISymbolSearchService.cs

```diff
@@ -66,37 +71,45 @@ protected PackageResult(string packageName, int rank)
         }
     }
 
-    internal class PackageWithTypeResult : PackageResult
+    [DataContract]
+    internal sealed class PackageWithTypeResult : PackageResult
     {
-        public readonly IList<string> ContainingNamespaceNames;
+        [DataMember(Order = 2)]
```

Comment:
oh interesting.  now i def want an analyzer :)

Reply:
Analyzer would be nice to have but not critical. We validate serializability of all types involved in RPC in a test. The test will fail if the data contract attributes are incorrect.

Reply:
great :)